vsintegration/src/FSharp.Editor/QuickInfo/Views.fs

```diff
@@ -82,26 +79,21 @@ module internal QuickInfoViewProvider =
                     currentContainerItems.Add(navigableTextRun :> obj)
                 | _ when item.Tag = LineBreak ->
                     flushRuns()
+                    // preserve succesive linebreaks
+                    if currentContainerItems.Count = 0 then
+                        runsCollection.Add(ClassifiedTextRun(classificationTag, System.String.Empty))
```

Comment:
Can `classificationTag` be changed to `PredefinedClassificationTypeNames.Other`?  Eventually we'll probably need to add tests for this and it will be easier if the classification tag is deterministic and not based on whatever we're currently wrapping, even if it's an empty string.